### PR TITLE
Documentation website

### PR DESCRIPTION
**Pull request by shd101wyy**
Example page: https://shd101wyy.github.io/iele-semantics/

Fixes: https://github.com/runtimeverification/iog-pm/issues/13

TODO:

* [ ] ~~Needs iele logo (light and dark) used in top bar~~ We decide to remove the the logo for now
* [ ] Discuss what we should be put in the sidebar
* [ ] ~~Where should the `Downloads` button in top bar point to~~ I removed this button
* [x] Needs to fix broken links in the generated HTML
* [ ] ~~Needs to support sytax highlighting of IELE and K (better using [Prism.js](https://prismjs.com/extending.html) instead of highlight.js)~~ This should be done in a new PR https://github.com/runtimeverification/iele-semantics/pull/209

### CODE REVIEW

**Comment by shd101wyy** [comment]
Hi @ehildenb , do you have any suggestions on what we should put in the side of the website? Right now we put `Homepage`, `Install`, `Design`. Thank you

**Comment by ehildenb** [comment]
@shd101wyy that seems like a good enough start! No need to make it too complicated. We can add documentation later.

But please also render all the K markdown files and link to them from teh home page.

**Comment by shd101wyy** [comment]
@ehildenb then I think this PR is ready for review. Thank you :)

File: Jenkinsfile [hunk 1]
```diff
@@ -46,5 +46,54 @@ pipeline {
...
+                  , message: "Deploy Phase Failed: ${env.BUILD_URL}"
+        }
+      }
+      environment {
+        AWS_ACCESS_KEY_ID     = credentials('aws-access-key-id')
+        AWS_SECRET_ACCESS_KEY = credentials('aws-secret-access-key')
+        AWS_REGION            = 'us-east-2'
+        GITHUB_TOKEN          = credentials('rv-jenkins')
+        GIT_SSH_COMMAND       = 'ssh -o StrictHostKeyChecking=accept-new'
+      }
```
Comment (ehildenb) [inline Jenkinsfile:73]:
```suggestion
```

File: Jenkinsfile [hunk 2]
```diff
@@ -46,5 +46,54 @@ pipeline {
...
+        dockerfile {
+          additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
+          reuseNode true
+        }
+      }
+      post {
+        failure {
+          slackSend color: '#cb2431'                                 \
+                  , channel: '#iele-internal'                                    \
+                  , message: "Deploy Phase Failed: ${env.BUILD_URL}"
```
Comment (ehildenb) [inline Jenkinsfile:64]:
```suggestion
          slackSend color: '#cb2431'                                 \
                  , channel: '#iele-internal'                        \
                  , message: "Deploy Phase Failed: ${env.BUILD_URL}"
```

**Review by ehildenb — APPROVED** [review]

File: Jenkinsfile [hunk 3]
```diff
@@ -46,5 +46,47 @@ pipeline {
...
+      when {
+        branch 'master'
+        beforeAgent true
+      }
+      agent {
+        dockerfile {
+          additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
+          reuseNode true
+        }
+      }
```
Comment (ehildenb) [inline Jenkinsfile:59]: @shd101wyy these lines shouldn't exist, because we're already in a a docker container (from the top of the Jenkinsfile).

**Pull request merged**